paymentapi/src/main/java/fi/hel/verkkokauppa/payment/paytrail/context/PaytrailPaymentContextBuilder.java

```diff
@@ -77,10 +82,20 @@ private PaytrailPaymentContext enrichWithNamespaceConfiguration(PaytrailPaymentC
 
     private PaytrailPaymentContext enrichWithMerchantConfiguration(PaytrailPaymentContext context, String merchantId) {
         MerchantDto merchantConfiguration = commonServiceConfigurationClient.getMerchantModel(merchantId, context.getNamespace());
-
         if (merchantConfiguration != null) {
-            ConfigurationParseUtil.parseConfigurationValueByKey(merchantConfiguration.getConfigurations(), ServiceConfigurationKeys.MERCHANT_SHOP_ID)
-                    .ifPresent(config -> context.setShopId(config.getValue()));
+            if (context.isUseShopInShop()) {
```

Comment:
isUseShopInShop => could it be written as canUseShopInShop ? :D

Reply:
The method is generated by lombok, I don't think it is necessary to create a custom method because of the naming because it is still quite understandable with "is" prefix.